# HG changeset patch
# User Rik <octave@nomad.inbox5.com>
# Date 1306379707 25200
#      Wed May 25 20:15:07 2011 -0700
# Branch stable
# Node ID e9e35a7077219242a923d091ec2fd7ee08b16196
# Parent  5dedac50fef955bfe0082796947b418973932a5c
doc: Add 'none' to list of plot linestyles.

* plot.txi: Add 'none' to list of plot linestyles.

diff --git a/doc/interpreter/plot.txi b/doc/interpreter/plot.txi
--- a/doc/interpreter/plot.txi
+++ b/doc/interpreter/plot.txi
@@ -2503,26 +2503,29 @@ one, or by name.  Recognized color names
 
 Line styles are specified by the following properties:
 
 @table @code
 @item linestyle
 May be one of
 @table @code
 @item "-"
-Solid lines.
+Solid line.  [default]
 
 @item "--"
-Dashed lines.
+Dashed line.
 
 @item ":"
-Points.
+Dotted line.
 
 @item "-."
 A dash-dot line.
+
+@item "none"
+No line.  Points will still be marked using the current Marker Style.
 @end table
 
 @item linewidth
 A number specifying the width of the line.  The default is 1.  A value
 of 2 is twice as wide as the default, etc.
 @end table
 
 @node Marker Styles
